12_listcomprehension_lab/01.py

```diff
@@ -1,8 +1,5 @@
-"""
-Use range() and list comprehension to get
-the list of all lowercase english letters
-Hint: look for chr() and ord()
-"""
-
-
-
+lst = []
+ascii = range(97, 123)
+for i in ascii:
+     lst.append([ x for x in chr(i)])
+print lst
```

Comment:
ascii_for_lower_case_letters = range(97, 123)
lst = [chr(i) for i in ascii_for_lower_case_letters]
print lst